packages/chart/src/components/SparkLine.js

```diff
@@ -90,6 +90,10 @@ export default function SparkLine({ width: parentWidth, height: parentHeight })
 
   const handleSparkLineTicks = [xScale.domain()[0], xScale.domain()[xScale.domain().length - 1]]
 
+  const isNumber = value => {
```

Comment:
Can you use the isNumber fn you lifted up to CdcChart here?

Reply:
Yes